packages/yourloops/components/patient/patient-row.tsx

```diff
@@ -277,7 +279,7 @@ const PatientRow: FunctionComponent<PatientRowProps> = ({ patient, filter }) =>
           >
             <Box>
               <IconActionButton
-                ariaLabel={`${t('remove-patient')}-${patient.profile.email}`}
+                ariaLabel={`${t('remove-patient')}-${email}`}
```

Comment:
opt: This aria label is not really correct. It should have a value similar to `Remove patient tim@diabeloop.fr`.
I write this comment as optional as it was already like that before.